### DISCUSSION TITLE
windivert install for CI

### DISCUSSION THREAD

**Discussion by chmorgan** [chmorgan/sharppcap#192] [General]
Hey @kayoub5. Do you know of any way to install windivert for CI use? I'm seeing a couple of odd behaviors, one is a crash somewhere in the windows tests, the other is a crash in the win-divert tests. Not sure how it didn't fail before but it also looks like the trailing --filter in azure-pipelines.yml was causing none of the filters to run. Removing that empty trailing --filter and now all of the azure tests are failing.

**Comment by chmorgan** [comment]
@kayoub5 and I just saw the installer in the win divert test code itself. Did you write that?

**Reply by kayoub5** [reply]
yes

**Reply by chmorgan** [reply]
It looks like the issue is that azure is using secure boot which prevents loading windivert dll. After disabling win divert the azure tests are passing.
